src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -91,10 +91,34 @@
     </ItemGroup>
   </Target>
 
+   <Target Name="_RazorCreateSourceHashFiles" DependsOnTargets="_RazorResolveSourceFiles">
+    <PropertyGroup>
+      <RazorSourcesHashFile>$(IntermediateOutputPath)$(MSBuildProjectName).RazorSourceHash.cache</RazorSourcesHashFile>
+    </PropertyGroup>
+
+    <Hash ItemsToHash="@(RazorCompile)">
```

Comment:
Is the idea that we hash the content or that we hash the file names? I expected that we were supposed to hash the file names, and used a separate up-to-date check for the contents?

Reply:
Ahh, I guess this is broken for file edits (I'll add a test). We'll need this in addition to `@(RazorCompile)`. This file would track renames \ deletes while `@(RazorCompile)` would track edits and additions.

Reply:
BTW we might end up using `RazorCompile` for the items that get passed to CSC (the generated `.cs` files). That's more consistent with the terms in the SDK